app/views/content/non-uk-teachers/train-to-teach-in-england-as-an-international-student.md

```diff
@@ -167,35 +174,39 @@ You can [find postgraduate teacher training courses](https://www.gov.uk/find-pos
 
 You can search by location, teaching subject and age group. You can also search for courses which pay a salary, and courses where student or skilled worker visa sponsorship is available.
 
-If you need a visa to train to teach in England, you must have a confirmed offer of a training place on a course offering student or skilled worker visa sponsorship before you apply for your visa.
+#### Choosing a subject to teach and preparing to apply
 
-Due to rules about immigration, not all providers can accept international applicants – contact them before you apply to check.
+The subject you apply to teach must usually be the same, or closely related to, the subject you studied at university. (For example, engineering graduates can apply to train to teach physics, but computer science graduates are unlikely to be accepted to train to teach geography.) 
+
+Your [teacher training adviser](#get-support-and-advice) can help with: 
+
+* your choice of subject
+* writing a personal statement
+* ways to research the [English education system](#learn-more-about-life-as-a-teacher-and-the-English-education-system)
```

Comment:
```suggestion
* ways to research the [English education system](#learn-more-about-life-as-a-teacher-and-the-english-education-system)
```

Reply:
@laurahermionetennant92 I think the capital E is affecting the jump link - works for me if it is changed to lower-case

Do you want to try that and see if it builds correctly now?